src/tests/test_utils/common_test_utils/src/node_builders/activation.cpp

```diff
@@ -45,6 +45,7 @@
 #include "openvino/op/swish.hpp"
 #include "openvino/op/tan.hpp"
 #include "openvino/op/tanh.hpp"
+#include "openvino/op/logical_not.hpp"
```

Comment:
Code style job failed. Please move this include upper and place after `#include "openvino/op/log.hpp"`. Clang format requires includes to be sorted.